tests/trainer/test_trainer.py

```diff
@@ -1763,27 +1763,6 @@ def backward(self, *args, **kwargs):
     ]
 
 
-def test_init_optimizers_resets_lightning_optimizers(tmpdir):
```

Comment:
Moved to `tests/core/test_lightning_optimizer.py` with no changes othger than the property access